### PR TITLE
Add suggestions for extra extensions

### PR DESCRIPTION
**Pull request by fabpot**

### CODE REVIEW

File: extra/twig-extra-bundle/TwigExtraBundle.php [hunk 1]
```diff
@@ -11,8 +11,16 @@
...
 
+use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
+use Twig\Extra\TwigExtraBundle\DependencyInjection\Compiler\MissingExtensionSuggestorPass;
 
 class TwigExtraBundle extends Bundle
 {
+    public function build(ContainerBuilder $container)
+    {
+        parent::build($container);
```
Comment (yceruto) [inline extra/twig-extra-bundle/TwigExtraBundle.php]: the parent doesn't have any operation, can it be removed?

Comment (fabpot) [reply extra/twig-extra-bundle/TwigExtraBundle.php]: I think we are doing the same in Symfony, just to be on the safe side if one day we add something in the parent?

**Comment by yceruto** [comment]
in TwigExtraExtension -> `new FileLocator(\dirname(__DIR__, 2).'/Resources/config')`

**Comment by yceruto** [comment]
or `new FileLocator(\dirname(__DIR__).'/Resources/config')` if you want to move it into `src/`

File: extra/twig-extra-bundle/src/TwigExtraBundle.php [hunk 2]
```diff
@@ -11,10 +11,19 @@
...
+    public function build(ContainerBuilder $container)
+    {
+        parent::build($container);
+
+        $container->addCompilerPass(new MissingExtensionSuggestorPass());
+    }
+
     public function getPath()
     {
         return \dirname(__DIR__);
```
Comment (yceruto) [inline extra/twig-extra-bundle/src/TwigExtraBundle.php]: if `src/Resources/config`, this should be removed, relying on the default (`src/`).

**Pull request merged**